### PR TITLE
refactor: use nuxt-class-component in example

### PR DESCRIPTION
**Pull request by clarkdo**

### CODE REVIEW

File: examples/vue-class-component/package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
     "nuxt": "latest",
-    "vue-class-component": "^5.0.1"
+    "nuxt-class-component": "^1.0.3",
+    "vue": "~2.4.2"
```
Comment (clarkdo) [inline examples/vue-class-component/package.json]: Can be remove after https://github.com/nuxt-community/nuxt-class-component/pull/3 merged and released

Comment (Atinux) [reply examples/vue-class-component/package.json]: I think you can remove it now @clarkdo :)

Comment (clarkdo) [reply examples/vue-class-component/package.json]: @Atinux It may need a version bump and publish:smile

Comment (clarkdo) [reply examples/vue-class-component/package.json]: @Atinux removed, can be merged after nuxt-class-component npm published

**Comment by clarkdo** [comment]
@Atinux removed, can be merged after nuxt-class-component npm published

**Comment by Atinux** [comment]
Hey @clarkdo 

I will publish 1.0.4 of nuxt-class-component and then merge this PR :)

**Comment by Atinux** [comment]
I really appreciate your help @clarkdo , and honestly, we really need help on nuxt-community and most of the modules has some issues, I will focus this weekend on making Nuxt.js 1.0.0-rca

**Comment by clarkdo** [comment]
@Atinux Thank you for the merge, and I will try to help with the nuxt-community modules issues. BTW, I saw we have conversations, but not very active, maybe it because I have different timezone with you, and I want to confirm do we have any other collaboration tool in common use?

**Comment by Atinux** [comment]
@clarkdo Could you send me an email so I can know you email and invite you to our Slack? :) You can see my public email on my GH profile

**Pull request merged**